tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -14,3 +14,19 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<button
+  class="mat-focus-indicator mat-icon-button mat-button-base"
+  mat-icon-button
+>
+  <span class="mat-button-wrapper">
+    <mat-icon
+      class="mat-icon mat-icon-no-color notranslate"
+      svgIcon="notifications_none_24px"
+      [matMenuTriggerFor]="menu"
```

Comment:
A button, or an interactive element, is a trigger, not a visual SVG. Put this on the `button`.

Reply:
Thanks! Done.